code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm

```diff
@@ -66,7 +76,6 @@
 //------------------------------------FALLING SUPPLY POD-------------------------------------//
 /obj/effect/temp_visual/DPfall
 	icon = 'icons/obj/2x2.dmi'
-	icon_state = "supplypod_falling"

```

Comment:
Keep this so the effect has a default icon, even if it's always overridden in Initialize